indico/modules/events/editing/client/js/editing/timeline/TimelineItem.jsx

```diff
@@ -84,8 +85,9 @@ export default function TimelineItem({block}) {
                   files={block.files}
                   downloadURL={block.downloadFilesURL}
                 />
-                {/* TODO: Check whether the current user is submitter */}
-                {isLastBlock && needsSubmitterConfirmation && <ChangesConfirmation />}
+                {isLastBlock && needsSubmitterConfirmation && canPerformSubmitterActions && (
```

Comment:
maybe `canPerformSubmitterAction` should be the first condition? I think it would be logically more correct (even though it doesn't change much)

Reply:
since it's not doing any function calls i don't care much about the order ;) but you are right, reversing the order of these conditions is more logical